docs/examples/1d_hist/1d_comparison_efficiency.py

```diff
@@ -0,0 +1,42 @@
+"""
+Efficiency comparison
+=====================
+
+Compare the ratio between two histograms h1 and h2 when the entries of h1 are a subset of the entries of h2.
+"""
+
+# Set the seed for the .sample() method
+import numpy as np
+
+np.random.seed(8311311)
```

Comment:
This function is a legacy function and should not be used anymore.

* https://numpy.org/doc/stable/reference/random/generated/numpy.random.seed.html